scripts/svm/executeRebalanceToSpokePool.ts

```diff
@@ -0,0 +1,293 @@
+// This script executes root bundle on HubPool that rebalances tokens to Solana Spoke Pool. Required environment:
+// - ETHERS_PROVIDER_URL: Ethereum RPC provider URL.
```

Comment:
As part of the tech debt task we can update this env vars to be consistent across the scripts so this comment is just an observation